.env.vagrant

```diff
@@ -20,6 +20,7 @@ FILE_TRANSFER_FEATURE=1
 
 KAFKA_BROKER_ADDR=chs-kafka:9092
 
+INQUIRY_EMAIL='enquiries@companieshouse.gov.uk'
```

Comment:
`ENQUIRY_EMAIL`?

Reply:
Did the same mistake in the chs configs :P